tests/samples/expected/ocaml.ml

```diff
@@ -573,6 +573,13 @@ let large_const =
     (long_argument_4 : int) ->
     val
 
+let _ =
+  [
+    1;
+    2;
+    3
```

Comment:
To the risk of belabouring a point. What I _actually_ expect is:

```suggestion
    1;
    2;
    3;
```

Reply:
I agree that's the ideal outcome, but we can't have it yet: see #199 
Should I add the issue to the v0.1 milestone?

Reply:
I added it to the milestone. We can remove it if the conclusion is that we shall not.

For the record: this should definitely be tackled in a separate PR either way. I was making that point because the PR is titled “List formatting”, which would seem to imply that there is no further issue with lists.